lib/client/hot.js

```diff
@@ -2,6 +2,10 @@
 
 const log = require('./log');
 
+// this is piped in at runtime build via DefinePlugin in /lib/plugins.js
+// eslint-disable-next-line no-unused-vars, no-undef
+const options = __hotClientOptions__;
```

Comment:
since this is already present in `lib/client/index.js`, we don't need to have webpack render that here too. that could cause some continuity issues down the road. instead:

- `module.exports = function update(currentHash) {` on line 100 of this (unaltered) file should be changed to `module.exports = function update(currentHash, options) {`>
- `function check() {` should be changed to `function check(options) {`
- `check();` should be called as `check(options);`
- `lib/client/index.js` line 33 should be changed to `update(currentHash, options);`

Reply:
Fixed this